### ISSUE TITLE
e2e tests for Java SDK

### ISSUE THREAD

**Issue by gkc** [atsign-foundation/at_java#24]
**Describe the bug**
no e2e tests at present 

**Expected behavior**
Tests of sharing data between at signs, and associated notification delivery and receipt. Tests should run as GitHub action on PR create / PR branch push / merge to trunk

**Comment by gkc** [comment]
Burned 5SP in PR41 setting up CI and writing tests. Left to do is configuring maven to treat unit tests and integration tests differently, then commit

**Comment by gkc** [comment]
Didn't get to this in PR42, moving to PR43

**Comment by JeremyTubongbanua** [comment]
@gkc For testing purposes, is there an atSign that we use and are okay with publishing the keys to GitHub?

**Comment by gkc** [comment]
No, we use GitHub secrets. I've got a branch with some integration tests that I need to merge which you can build on

**Comment by gkc** [comment]
Will close out early this week in PR44

**Comment by JeremyTubongbanua** [comment]
> No, we use GitHub secrets. I've got a branch with some integration tests that I need to merge which you can build on

Gentle ping

**Comment by gkc** [comment]
No progress during PR44 due to time constraints. Will be my first priority in PR45.

**Comment by gkc** [comment]
Moving to backlog until Java-using customers rev up again
